lib/sda/overlay.go

```diff
@@ -28,17 +28,26 @@ type Overlay struct {
 	entityListLock sync.Mutex
 	// cache for relating token(~Node) to TreeNode
 	cache TreeNodeCache
+
+	// TreeNodeInstance part
+	instances     map[TokenID]*TreeNodeInstance
+	instancesInfo map[TokenID]bool
```

Comment:
Can we name it `instancesSentInfo`? It will be `false` by default, then you can set it to `true` once the info has been sent.